server/handler/fetcher.go

```diff
@@ -179,3 +197,12 @@ func (cf *ConfigFetcher) unmarshalConfig(bytes []byte) (*policy.Config, error) {
 
 	return &config, nil
 }
+
+func (cf *ConfigFetcher) hasTooLargeError(errorResponse *github.ErrorResponse) bool {
```

Comment:
style: I'd call this `isTooLargeError` and make it a plain non-exported function instead of attaching it to `ConfigFetcher`

Reply:
done